js/electron.js

```diff
@@ -47,7 +47,7 @@ function createWindow() {
 
 	// and load the index.html of the app.
 	//mainWindow.loadURL('file://' + __dirname + '../../index.html');
-	mainWindow.loadURL("http://localhost:" + config.port);
+	mainWindow.loadURL(`http://${config.address}:${config.port}`);
```

Comment:
Or did you mean to say to not use templated strings at all? I feel it improves the readability of the code, but if it's against the coding guidelines for this project, I have no problem changing it.